testing/trafficserver/APKBUILD

```diff
@@ -0,0 +1,162 @@
+# Contributor: tcely <trafficserver+aports@tcely.33mail.com>
+# Maintainer: tcely <trafficserver+aports@tcely.33mail.com>
+_upstream="trafficserver"
+pkgname="${_upstream}"
```

Comment:
Why not directly define pkgname as trafficserver?

Reply:
The package name is more likely to change than the upstream project name.